src/js/views/patients/patient/patient.scss

```diff
@@ -150,3 +150,7 @@
   vertical-align: middle;
   width: 34px;
 }
+
+.patient__flow-status-icon {
```

Comment:
Can we look at the two other icons with no button on the lists and come up with a solution that works for all of them? `.workflows__flow-status` and `.flow-action--attachment`  Also it'd be great to stick to 8px or 4px increments, so this'd need to be `padding: 0 8px;` with a 1px border that's transparent if this is the best solution for all the places.